docs/hpc/applications/guides/python.md

```diff
@@ -155,3 +155,83 @@ We can see when extra memory was allocated and when it was cleared. This is clea
 ## Running Python code from within Matlab
 
 If you would like to run your Python code from within Matlab so that you can make use of features available in both the Matlab and Python, you can do so easily. We have provided a small example here on [How to run Python code from within Matlab](./matlab.md#running-python-code-from-within-matlab).
+
+## Running Matlab code from within Python
+
+If you would like to use some of the Matlab's function inside Python code, you can do so with the help of [`Matlab Engine`](https://www.mathworks.com/help/matlab/matlab_external/install-the-matlab-engine-for-python.html) package. This allows you to start a Matlab session and run Matlab commands from Python.
+
+In many cases, people would like to use many of the packages in their conda environment along with the ability to run the matlab code as well. We explain the detailed steps below.
+
+First, you would need to load an appropriate Matlab module. In our case, we use the following.
+
+```bash
+$ module load MATLAB/2024b
+```
+
+Since, we will be installing the `MATLAB Engine` in our own environment, you would need to make sure that the folder `glnxa64` from `MATLAB` is in your `LD_LIBRARY_PATH`. You can do that by
```

Comment:
MATLAB is capitalised but you have it as Matlab else where.

Reply:
We will use capital MATLAB everywhere to be consistent with Mathworks and this page itself.